### ISSUE TITLE
karma: gulp test:auto doesn't work (1.0.0-rc1)

### ISSUE THREAD

**Issue by eseceve** [Swiip/generator-gulp-angular#777]
So, `gulp test` works fine but `gulp test:auto` doesn't recognize tests.

![image to my console](http://i.imgur.com/dUGN4hW.png)
> http://i.imgur.com/dUGN4hW.png

.yo-rc
```
{
  "generator-gulp-angular": {
    "version": "1.0.0-rc1",
    "props": {
      "angularVersion": "~1.4.2",
      "angularModules": [
        {
          "key": "animate",
          "module": "ngAnimate"
        },
        {
          "key": "cookies",
          "module": "ngCookies"
        },
        {
          "key": "touch",
          "module": "ngTouch"
        },
        {
          "key": "sanitize",
          "module": "ngSanitize"
        },
        {
          "key": "messages",
          "module": "ngMessages"
        },
        {
          "key": "aria",
          "module": "ngAria"
        }
      ],
      "jQuery": {
        "key": "jqLite"
      },
      "resource": {
        "key": "$http",
        "module": null
      },
      "router": {
        "key": "ui-router",
        "module": "ui.router"
      },
      "ui": {
        "key": "bootstrap",
        "module": null
      },
      "bootstrapComponents": {
        "key": "ui-bootstrap",
        "module": "ui.bootstrap"
      },
      "cssPreprocessor": {
        "key": "node-sass",
        "extension": "scss"
      },
      "jsPreprocessor": {
        "key": "babel",
        "extension": "js",
        "srcExtension": "es6"
      },
      "htmlPreprocessor": {
        "key": "noHtmlPrepro",
        "extension": "html"
      },
      "foundationComponents": {
        "name": null,
        "version": null,
        "key": null,
        "module": null
      },
      "advancedFeatures": [
        "imagemin"
      ],
      "paths": {
        "src": "src",
        "dist": "dist",
        "e2e": "e2e",
        "tmp": ".tmp"
      }
    }
  }
}
```

**Comment by Swiip** [comment]
Reproduced... I look into it.

**Comment by Swiip** [comment]
I found that the coverage is always used. It breaks the watch mode. I fixed it here: #783
